src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -829,7 +829,7 @@ protected SavedPending SavePending()
             var result = new SavedPending(ref _pendingBranches, ref _labelsSeen);
             if (_trackExceptions)
             {
-                _pendingBranches.Add(new PendingBranch(null, this.State.Clone()));
```

Comment:
Many other similar cases were missing the "Clone()", so I have moved it into the ctor.
I most cases the "Clone()" is a no-op. Only for large dataflow states with 32+ locals there is a difference.